### PR TITLE
Sprint 32

### PR DESCRIPTION
**Pull request by ericsorenson**
# Sprint 32

Blue / Green deployments are part of this sprint. Please see docs/BLUE_GREEN.md for more details.

## [1584 High Priority Indicator for External Filings (automated)](https://github.com/flexion/ef-cms/issues/1584)
image

## [1193 Notice of Trial Issued](https://github.com/flexion/ef-cms/issues/1193)
image

## [3922 Seal a case - Make entire case hidden from public view](https://github.com/flexion/ef-cms/issues/3922)
image

## [3573 Court User: Unlink Consolidated Cases](https://github.com/flexion/ef-cms/issues/3573)
image

### CODE REVIEW

**Comment by waldoj** [comment]
### PR Review Checklist

- [ ] begin by deleting any existing copy of the Git repository, running `docker system prune` to remove any cached artifacts from any prior code review
- [ ] clone a fresh copy of the Git repository (`git clone https://github.com/ustaxcourt/ef-cms.git`), fetch the pull request and name it for the sprint (e.g., `git fetch origin pull/204/head:sprint-43`), and then switch to that branch (e.g. `git checkout sprint-43`)
- [ ] review the pull request itself, to get oriented
	- [ ] read the description of the pull request, which should summarize the changes made
	- [ ] read through every task on the Scrum board that's encompassed by this pull request
	- [ ] read the description of the commits that comprise the pull request
- [ ] run all tests and generate test data with `./test-all.sh`
- [ ] stand up the site locally, with `./docker-run.sh`
	- [ ] test each new story in all major browsers
		- [ ] for internal Court functionality, perform the most thorough testing in Chrome, though also test in Edge and Firefox
		- [ ] for public-facing functionality, test in browsers consistent with [public browser use data](https://analytics.usa.gov/), including Internet Explorer 11
		- [ ] test in Mobile Safari and Mobile Chrome, with the caveat that not all internal Court functionality will be necessary on these platforms
	- [ ] use an automated audit tool for code quality and practices (recommended: [Chrome DevTools](https://developers.google.com/web/tools/chrome-devtools/), aka [Lighthouse](https://chrome.google.com/webstore/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk))
		- [ ] look at efficiency of page loads, asset sizes, HTTP connection management, etc.
	- [ ] review for accessibility
		- [ ] use an automated audit tool, such as Chrome Audit or aXe
		- [ ] navigate site only with the keyboard
		- [ ] use VoiceOver or Narrator to navigate the site with audio only, with the display turned off
		- [ ] manually test anything that pa11y cannot test automatically (e.g., contrast of text over images)
- [ ] [review static code analysis results](https://sonarcloud.io/organizations/flexion-github/projects) in the vendor’s SonarCloud environment
- [ ] examine OWASP ZAP output in `docs/`, to ensure that any errors are known to be false positives or have been previously declared to be acceptable
- [ ] skim all new code, in the context of existing code, looking for problems (knowing that the vast majority of new code will be covered by tests)
- [ ] review all tests
	- [ ] look at code coverage of tests in [the vendor's SonarCloud environment](https://sonarcloud.io/organizations/flexion-github/projects)
		- [ ] review untested code rigorously
	- [ ] methodically review all new tests for correctness, quality of naming
- [ ] review documentation to ensure that it matches changes
- [ ] provide comments on the pull request on GitHub, as necessary
	- [ ] for comments that are specific to a particular line of code, comment on those specific lines
	- [ ] for comments that are more general, attach the comment to a random line in `README.md` (as opposed to commenting on the pull request itself), to be able to use GitHub's ability to thread discussions on those comments
- [ ] for each feature-level bug (i.e., it’s working as designed, but designed wrong), open a new issue and put it in the backlog

File: web-client/integration-tests/journey/docketClerkVerifiesDocketEntryMetaUpdates.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+export default (test, docketRecordIndex = 1) => {
+  return it('docket clerk verifies docket entry meta update', async () => {
+    // FIXME: commenting this out for now until the form page is complete
```
[the remark this answers is not in the document]
Comment (rachelschneiderman) [reply web-client/integration-tests/journey/docketClerkVerifiesDocketEntryMetaUpdates.js]: Correct. This was a story that was in progress when the sprint ended. The complete feature will be in the PR for sprint 33.

**Comment by waldoj** [comment]
@JessicaMarine, pending a successful build in CircleCI, I believe this pull request is ready to be merged into `staging`.
